packages/forms-web-app/src/pages/examination/_translations/cy.json

```diff
@@ -0,0 +1,4 @@
+{
+	"yes": "Oes",
+	"no": "Nac oes"
+}
```

Comment:
I wonder if it could be worth putting these in the common translations file, since these ones are likely to be used elsewhere too.

Reply:
Good idea, thanks Hadrian.